### ISSUE TITLE
Telegraf service crashes with "Panic: Runtime Error: Slice bounds out of range"

### ISSUE THREAD

**Issue by deveth0** [influxdata/telegraf#2385]
### Bug report

__System info:__ [Include InfluxDB version, operating system name, and other relevant details]
Debian 8 running Telegraf 1.2.0-1 from repos.influxdata.com

__Steps to reproduce:__
-

__Expected behavior:__ 
Telegraf does not crash

__Actual behavior:__ 
After some time the telegraf service crashes with the attached error. I'm not sure if this is related to the snmp errors we see but I'd expect that telegraf just ignores those.

__Additional info:__ 
```
2017-02-07T08:25:41Z E! ERROR: input [inputs.snmp] took longer to collect than collection interval (10s)
2017-02-07T08:26:00Z E! ERROR: input [inputs.snmp] took longer to collect than collection interval (10s)
2017-02-07T08:26:20Z E! ERROR: input [inputs.snmp] took longer to collect than collection interval (10s)
panic: runtime error: slice bounds out of range

goroutine 3522 [running]:
panic(0xf2a2e0, 0xc4200100e0)
        /usr/local/go/src/runtime/panic.go:500 +0x1a1
github.com/influxdata/telegraf/metric.(*metric).Fields(0xc420911780, 0xc420261110)
        /home/ubuntu/telegraf-build/src/github.com/influxdata/telegraf/metric/metric.go:279 +0x5dd
github.com/influxdata/telegraf/metric.(*metric).Point(0xc420911780, 0xc42071ac20)
        /home/ubuntu/telegraf-build/src/github.com/influxdata/telegraf/metric/metric.go:141 +0x80
github.com/influxdata/telegraf/plugins/outputs/influxdb.(*InfluxDB).Write(0xc420492100, 0xc42064a000, 0x7e, 0x7e, 0xc4208b0e58, 0x6b3911)
        /home/ubuntu/telegraf-build/src/github.com/influxdata/telegraf/plugins/outputs/influxdb/influxdb.go:194 +0x154
github.com/influxdata/telegraf/internal/models.(*RunningOutput).write(0xc4202086e0, 0xc42064a000, 0x7e, 0x7e, 0x7e, 0xc4208b0f28)
        /home/ubuntu/telegraf-build/src/github.com/influxdata/telegraf/internal/models/running_output.go:170 +0xa1
github.com/influxdata/telegraf/internal/models.(*RunningOutput).Write(0xc4202086e0, 0x11562e0, 0xc420441040)
        /home/ubuntu/telegraf-build/src/github.com/influxdata/telegraf/internal/models/running_output.go:154 +0x49c
github.com/influxdata/telegraf/agent.(*Agent).flush.func1(0xc420441040, 0xc4202086e0)
        /home/ubuntu/telegraf-build/src/github.com/influxdata/telegraf/agent/agent.go:232 +0x68
created by github.com/influxdata/telegraf/agent.(*Agent).flush
        /home/ubuntu/telegraf-build/src/github.com/influxdata/telegraf/agent/agent.go:237 +0xb4

```

**Comment by sparrc** [comment]
this has been fixed in 1.2.1, please upgrade

**Comment by deveth0** [comment]
Cool, thank you! Could not find it in the changelog :)
